src/Shared/Traits.cs

```diff
@@ -115,6 +115,11 @@ internal class EscapeHatches
         /// </summary>
         public readonly bool AlwaysUseContentTimestamp = Environment.GetEnvironmentVariable("MSBUILDALWAYSCHECKCONTENTTIMESTAMP") == "1";
 
+        /// <summary>
+        /// Stop truncating task inputs when logging them. Setting any value disables this.
```

Comment:
```suggestion
        /// Truncate task inputs when logging them. This can reduce memory pressure
        /// at the expense of log usefulness.
```